collector/database/settings.php

```diff
@@ -28,6 +28,18 @@
 use cltr_database\output\admin_setting_days_configduration;
 
 if ($hassiteconfig) {
+    $ADMIN->add('reports', new admin_category('cltr_database_reports', get_string('pluginname', 'cltr_database')));
```

Comment:
What other reports should live in this? If its just the 1, it would be better to just add it directly to the 'reports' category

Reply:
done